design/docs-reorg.md

```diff
@@ -0,0 +1,73 @@
+# Documentation Reorganization
+
+This document proposes a new outline for the Velero documentation that is intended to be easier for both users and contributors to interact with.
+
+## Goals
+
+- Agree on the desired outline (sections and pages) for Velero documentation.
+
+## Non Goals
+
+- Write/re-write all new/revised pages (individual page writes/rewrites to be tracked as individual issues).
+
+## Background
+
+The Velero documentation has evolved organically since the project's inception.
+While it contains a lot of useful information, it is not currently organized in a way that is easy to navigate for users or contributors.
+For example, the introductory section jumps straight into a discussion of Velero's architecture, rather than starting with a good overview of its core features.
+Also, the documentation on how to actually *use* Velero is fairly superficial, only covering the most straight-forward use cases.
+Additionally, different types of information are often mixed together (i.e. conceptual overviews along with detailed setup instructions).
+
+This design document proposes a new outline for the documentation that is intended to be more easily navigated by both users and contributors.
+The new outline both reorganizes the existing content, *and* calls for new content where relevant.
+
+## High-Level Design
+
+- Concepts
+    - Overview
+        - Core Features
+        - Architecture
+    - Storage Providers/Plugins
```

Comment:
I might split these up, or perhaps add a new page - while the storage providers have been the most important plugins usually, there are also the ItemAction plugins.

That plugin page that I'm thinking of may, however, be out of scope for this particular redesign, as I envision it to be something that discusses the types of plugins that Velero supports, their APIs, and possibly how to write one.